android/libs.versions.toml

```diff
@@ -0,0 +1,37 @@
+[versions]
+# Android versions
+minSdk = "21"
+targetSdk = "34"
+compileSdk = "34"
+buildTools = "34.0.0"
+# Dependencies versions
+agp = "8.1.1"
+download = "5.4.0"
+flipper = "0.201.0"
+fresco = "3.1.0"
+kotlin = "1.8.0"
+nexus-publish = "1.3.0"
+# Native Dependencies
+boost="1_83_0"
+doubleconversion="1.1.6"
+fmt="6.2.1"
+folly="2022.05.16.00"
+glog="0.3.5"
+libevent="2.1.12"
+gtest="1.12.1"
+
+[libraries]
+android-gradle-plugin = { module = "com.android.tools.build:gradle", version.ref = "agp" }
+
+fresco = { module = "com.facebook.fresco:fresco", version.ref = "fresco" }
+fresco-middleware = { module = "com.facebook.fresco:middleware", version.ref = "fresco" }
```

Comment:
nit: can remove some libs here because we read the versions from toml inside react-native-lab.

Reply:
Removed